kmath-geometry/src/jvmTest/kotlin/scientifik/kmath/geometry/Examples.kt

```diff
@@ -0,0 +1,64 @@
+package scientifik.kmath.geometry
+
+import scientifik.kmath.dimensions.D4
+import scientifik.kmath.geometry.transformations.mapTo
+import scientifik.kmath.geometry.transformations.projection
+import scientifik.kmath.geometry.transformations.shiftBy
+import scientifik.kmath.operations.RealField
+import scientifik.kmath.operations.invoke
+
+fun main2() {
+    (EuclideanSpace(D4, RealField)) {
```

Comment:
Should use invoke, it will be better readable.

Reply:
invoke is used

Reply:
I mean explicit invoke. It allows to avoid braces. But it is a cosmetic change. We should consider removing operator the to avoid the confusion.